### DISCUSSION TITLE
Bug hunting

### DISCUSSION THREAD

**Discussion by Oboe831** [Oboe831/DragonBallRedux#3] [Q&A]
<h2>Please tell me of any bugs or errors you find with the sheet in this discussion!</h2>

**Comment by Jestyyy** [comment]
So the sheet is pretty unusable as it is with the displacement and confusion with building up the sheet I was wondering if you ever plan on updating it? I really like the idea of it and what the sheet looked like in sheet selection, hope to hear about it!

**Reply by Oboe831** [reply]
Hi, yes I do plan to update the sheet. I am actually in the middle of making a third version of the sheet, but then roll20 decided to change how its code library works and I've just been fixing the bugs in the next version, I thought that the this version would be fine as it should still be using the legacy version!

**Reply by Oboe831** [reply]
I just found the issue and it is now fixed. Come Monday when they update the sheets, it will now be incorporated into roll20 sheet and should be fixed.

**Reply by Jestyyy** [reply]
You are a champ thank you so much! Looking forward to the third version in the future as well.
